srcpkgs/startup/template

```diff
@@ -0,0 +1,66 @@
+# Template file for 'startup'
+pkgname=startup
+version=1.13.3
+revision=1
+build_style=gnu-configure
+configure_args="--disable-sysvcompat --disable-dconf-bridge --disable-cgroups"
+hostmakedepends="autoconf automake libtool pkg-config gettext-devel xz nih-devel"
+makedepends="libnih-devel eudev-libudev-devel dbus-devel json-c-devel"
+checkdepends="python3 dbus"
+short_desc="Event driven task and service manager"
+maintainer="Cameron Nemo <camerontnorman@gmail.com>"
+license="GPL-2.0-only"
+homepage="https://gitlab.com/chinstrap/startup/startup"
+changelog="${homepage}/raw/master/NEWS"
+distfiles="${homepage}/-/archive/${version}/startup-${version}.tar.gz"
+checksum=b63de554d039db6cc920a50a4d38f83529c362ef725dece557247524a84d78ae
+
+make_dirs="/var/log/upstart 0755 root root"
+
+pre_configure() {
+	autoreconf -fi
+}
+
+startup-tools_package() {
+	depends="${sourcepkg}>=${version}_${revision} python3-dbus python3-gobject"
+	short_desc+=" - monitoring and analysis tools"
+	pkg_install() {
+		vmove usr/bin/initctl2dot
+		vmove usr/bin/startup-monitor
+		vmove usr/share/icons
+		vmove usr/share/startup/icons
+		vmove usr/share/applications/startup-monitor.desktop
+		vmove usr/share/man/man8/startup-monitor.8
+	}
+}
+
+startup-udev-bridge_package() {
```

Comment:
why have udev-bridge separated from other bridges ?

Reply:
libudev dependency. All the other bridges introduce no new dependencies. If the dconf bridge is enabled later I intend to make it a subpackage.

Reply:
libudev is already part of base-system.